### PR TITLE
Modified Godot games for muOS, fixed old ports to use runtimes, made ports use included xdelta3.

### PR DESCRIPTION
**Pull request by kloptops**

### CODE REVIEW

File: ports/megaman/Megaman.sh [hunk 1]
```diff
@@ -59,7 +62,7 @@ export FRT_NO_EXIT_SHORTCUTS=FRT_NO_EXIT_SHORTCUTS
 
 $ESUDO chmod 666 /dev/uinput
 $GPTOKEYB "$runtime" -c "./a.gptk" &
-"$runtime" -path "data.pck" 2>&1 | $ESUDO tee -a ./log.txt
+"$runtime" $GODOT2_OPTS -path "data.pck" 2>&1 | $ESUDO tee -a ./log.txt
```
Comment (tabreturn) [inline ports/megaman/Megaman.sh:65]: GODOT2_OPT or GODOT2_OPTS ?

File: ports/spooky/Spooky.sh [hunk 2]
```diff
@@ -64,7 +67,7 @@ export FRT_NO_EXIT_SHORTCUTS=FRT_NO_EXIT_SHORTCUTS
 
 $ESUDO chmod 666 /dev/uinput
 $GPTOKEYB "$runtime" -c "spooky.gptk" &
-"$runtime" -path "data.pck"
+"$runtime" $GODOT2_OPT -path "data.pck"
```
Comment (tabreturn) [inline ports/spooky/Spooky.sh:70]: GODOT2_OPT or GODOT2_OPTS ?

**Review by tabreturn — COMMENTED** [review]
Looks good (and a fair chunk of work!). I spotted one issue (the GODOT2 thing) that I've noted. I'm guessing the next step is to remove the xdelta lib files that are no longer necssary.

**Review by tabreturn — APPROVED** [review]

**Review by Cebion — APPROVED** [review]
👍

**Pull request merged**